crates/core_simd/src/ops.rs

```diff
@@ -96,8 +96,23 @@ macro_rules! int_divrem_guard {
                 // Nice base case to make it easy to const-fold away the other branch.
                 $rhs
             };
-            // Safety: $lhs and rhs are vectors
-            unsafe { core::intrinsics::simd::$simd_call($lhs, rhs) }
+
+            // aarch64 div fails for arbitrary `v % 0`, mod fails when rhs is MIN, for non-powers-of-two
+            // these operations aren't vectorized on aarch64 anyway
```

Comment:
These are LLVM backend bugs, right? simd_div/simd_rem still *should* work the same on all targets?

That seems worth tracking somewhere, having subtly buggy intrinsics is no good.

Reply:
also, theoretically LLVM should be able to generate SIMD code for division/remainder by a constant, by using the exact same fancy math as it would use for scalars (which it unfortunately currently does after scalarization of div ops), so once LLVM's bugs are fixed, I think we should switch back to generating SIMD ops.

Reply:
Yes, these are definitely backend bugs